corehq/apps/tzmigration/timezonemigration.py

```diff
@@ -47,20 +50,24 @@ def _json_diff(obj1, obj2, path, track_list_indices=True):
         return
     elif Ellipsis in (obj1, obj2):
         yield FormJsonDiff('missing', path, obj1, obj2)
+    elif isinstance(obj1, dict):
+        if not isinstance(obj2, dict):
+            # special case to deal with OrderedDicts
+            yield FormJsonDiff('type', path, obj1, obj2)
```

Comment:
In Python 3.6 and later all dicts preserve ordering. Will that matter here?

Reply:
That's fine and won't be an issue here. This was only an issue when there's a mix of `dict` and `OrderedDict`